src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -38,10 +39,36 @@ const styles = {
   }),
 };
 
-function createTitle(header: RevisionsHeader) {
-  return header.test === '' || header.suite === header.test
-    ? header.suite
-    : `${header.suite} ${header.test}`;
+function createTitle(
+  header: RevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  const isTestUnavailable = header.test === '' || header.suite === header.test;
```

Comment:
```suggestion
  if (header.suite === header.test)
     header.test = '';
```
I believe doing this will allow you to remove some extra code.

Reply:
The suggestion seems to achieve something other than the line that's already there. Can you elaborate more?

Reply:
For now ignore this comment as it doesn't impact the purpose of your ticket.